### ISSUE TITLE
Keycloak Not working with F5 configured with https

### ISSUE THREAD

**Issue by gopalkrishnams** [keycloak/keycloak#26696]
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

admin/ui

### Describe the bug

we have configured keycloak with reverse proxy(F5) with https. upon login to the admin screen, the admin UI gets into an infinite loop with the following error. I have configured frontenturl but still same issue. Please help
Error: 
org.keycloak.jose.jws.JWSInputException: java.lang.IllegalArgumentException: Parsing error
	at org.keycloak.jose.jws.JWSInput.<init>(JWSInput.java:59)
	at org.keycloak.jose.jws.DefaultTokenManager.decode(DefaultTokenManager.java:90)
	at org.keycloak.protocol.oidc.TokenManager.toRefreshToken(TokenManager.java:527)
	at org.keycloak.protocol.oidc.TokenManager.verifyRefreshToken(TokenManager.java:478)
	at org.keycloak.protocol.oidc.TokenManager.refreshAccessToken(TokenManager.java:384)
	at org.keycloak.protocol.oidc.endpoints.TokenEndpoint.refreshTokenGrant(TokenEndpoint.java:567)
	at org.keycloak.protocol.oidc.endpoints.TokenEndpoint.processGrantRequest(TokenEndpoint.java:209)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.jboss.resteasy.core.MethodInjectorImpl.invoke(MethodInjectorImpl.java:154)
	at org.jboss.resteasy.core.MethodInjectorImpl.invoke(MethodInjectorImpl.java:118)
Caused by: java.lang.IllegalArgumentException: Parsing error
	at org.keycloak.jose.jws.JWSInput.<init>(JWSInput.java:46)
	... 76 common frames omitted

2024-02-01T11:52:15.489Z  WARN 1015544 --- [http-nio-8096-exec-5] org.keycloak.events                      : type=REFRESH_TOKEN_ERROR, realmId=e21ea005-29f7-44d0-a4d7-fd23dae89654, clientId=security-admin-console, userId=null, ipAddress=180.179.1.228, error=invalid_token, grant_type=refresh_token, client_auth_method=client-secret
 

### Version

23.0.1

### Expected behavior

admin UI should load after login with reverse proxy configured with https

### Actual behavior

admin ui gets into infinite loop after login. 

### How to Reproduce?

Configure reverse proxy with https pointing to keycloak server. Access Login url  via reverse proxy.

**Comment by jonkoops** [comment]
Duplicate of #14666

**Comment by gopalkrishnams** [comment]
none of the suggestions mentioned in the ticket is working.  We have Keycloak embedded within the spring boot app. Please help us to resolve

**Comment by mma-muzammil** [comment]
We observed a similar issue with F5, reason being F5 creating AUTH_SESSION_ID cookie on the first request itself, whereas keycloak creates AUTH_SESSION_ID cookie after authentication. Conflicting with two AUTH_SESSION_ID cookies. We resolved it by following below article, configuring F5 for passive mode cookies, F5 only "listens" for the AUTH_SESSION_ID cookie it must never creates it.

https://my.f5.com/manage/s/article/K83419154
